src/ray/core_worker/task_manager.h

```diff
@@ -139,6 +156,14 @@ class TaskManager : public TaskFinisherInterface {
   /// Protects below fields.
   mutable absl::Mutex mu_;
 
+  struct ActorTaskState {
```

Comment:
Could you document the struct? Also, I slightly prefer `num_submitted` over `submitted` to stay consistent with other places where we use `num_*`.